plugins/qlcplus_4.11.2/export.js

```diff
@@ -297,7 +297,7 @@ function addHeads(xmlMode, mode) {
  * @returns {!string} The first of the fixture's categories that is supported by QLC+, defaults to 'Other'.
  */
 function getFixtureType(fixture) {
-  const ignoredCats = [`Blinder`, `Matrix`];
+  const ignoredCats = [`Blinder`, `Matrix`, `Pixel Bar`, `Stand`];
 
   for (const category of fixture.categories) {
```

Comment:
could be refactored into ``return fixture.categories.find(cat => !ignoredCats.includes(cat)) || `Other`;``